config

```diff
@@ -271,6 +274,9 @@ bindsym $mod+$alt+Ctrl+$ws7_key move container to workspace number $ws17; worksp
 bindsym $mod+$alt+Ctrl+$ws8_key move container to workspace number $ws18; workspace number $ws18
 bindsym $mod+$alt+Ctrl+$ws9_key move container to workspace number $ws19; workspace number $ws19
 
+## Modify // Move Window to Next Free Workspace // <><Shift> ` ##
+bindsym $mod+shift+grave exec --no-startup-id /usr/share/i3xrocks/next-workspace --startnum 1 --move-window-and-follow
```

Comment:
I suggest `$mod+alt+grave` as that would be more in line with moving windows to workspaces (e.g. `bindsym $mod+$alt+0 move container to workspace number $ws0; workspace number $ws0
`)

Reply:
Great catch!